Resources/Textures/_Nuclear14/Structures/Wallmounts/walldecor.rsi/meta.json

```diff
@@ -99,6 +99,15 @@
     },
     {
       "name": "poster_wecan"
+    },
```

Comment:
If you've created these sprites they'll either need to go in their own meta.json file with your copyright, or append your copyright info to this one. E.g. "X,Y,Z created by Marine...."